### PR TITLE
[Task] Increment project version for release

### PR DESCRIPTION
**Pull request by Leah-Xia-Microsoft**
## Purpose

* Increase project version in Xcode to 1.1.0-beta.1

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe:
    - Project settings update
```

### CODE REVIEW

File: AzureCommunicationUI/AzureCommunicationUIDemoApp/AzureCommunicationUIDemoApp.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -744,7 +744,7 @@
 					"@executable_path/Frameworks",
 				);
 				MARKETING_VERSION = 1.0;
-				PRODUCT_BUNDLE_IDENTIFIER = com.azure.ios.communication.ui.meetings.AzureCommunicationUIDemoApp;
+				PRODUCT_BUNDLE_IDENTIFIER = com.microsoft.azure.communication.ui.demoapp;
```
Comment (AmyL219) [inline AzureCommunicationUI/AzureCommunicationUIDemoApp/AzureCommunicationUIDemoApp.xcodeproj/project.pbxproj:747]: Should we update the bundle id here?

Comment (jimchou-dev) [reply AzureCommunicationUI/AzureCommunicationUIDemoApp/AzureCommunicationUIDemoApp.xcodeproj/project.pbxproj:747]: The bundle identifier needs to match the provisioning profile, probably better to change it here than modifying during the pipeline.

**Review by JoshuaLai — DISMISSED** [review]

**Comment by JoshuaLai** [comment]
I think there are other parts in the read me / documentation (like the code snippets) that will need the version bumped as well.

File: AzureCommunicationUI/AzureCommunicationUIDemoApp/AzureCommunicationUIDemoApp.xcodeproj/project.pbxproj (hunk 1 above)
Comment (AmyL219) [reply AzureCommunicationUI/AzureCommunicationUIDemoApp/AzureCommunicationUIDemoApp.xcodeproj/project.pbxproj:747]: We have multiple bundle ids, correct? Is this bundle id for MS App Center or Apple App Store?

Comment (jimchou-dev) [reply AzureCommunicationUI/AzureCommunicationUIDemoApp/AzureCommunicationUIDemoApp.xcodeproj/project.pbxproj:747]: You're right,  there is different bundle ID for dogfood (app center) vs production (app store)

@jpeng-ms lets modify the bundle identifier as part of pipeline.

Comment (jpeng-ms) [reply AzureCommunicationUI/AzureCommunicationUIDemoApp/AzureCommunicationUIDemoApp.xcodeproj/project.pbxproj:747]: Thanks I have created a task for this. @Leah-Xia-Microsoft I guess we could revert this and let pipeline to handle it now

**Comment by Leah-Xia-Microsoft** [comment]
> I think there are other parts in the read me / documentation (like the code snippets) that will need the version bumped as well.

I see there's a separate task for read me/documentation update. Do you think I should include those update in this PR or leave it for the read me/ documentation task? (John's taking that one)

**Comment by jpeng-ms** [comment]
> > I think there are other parts in the read me / documentation (like the code snippets) that will need the version bumped as well.
> 
> I see there's a separate task for read me/documentation update. Do you think I should include those update in this PR or leave it for the read me/ documentation task? (John's taking that one)

yea, I am handling it. Posted a discussion in teams channel, but got no response yet. But I guess I will create a PR later.

**Review by JoshuaLai — APPROVED** [review]

**Review by palatter — APPROVED** [review]

**Pull request merged**